tests/apollo/test_skvbc_preexecution.py

```diff
@@ -156,12 +157,8 @@ async def test_view_change(self, bft_network):
 
         clients = bft_network.clients.values()
         client = random.choice(list(clients))
-        key_before_vc = skvbc.random_key()
-        value_before_vc = skvbc.random_value()
-        write_set = [(key_before_vc, value_before_vc)]
 
-        await self.send_single_write_with_pre_execution_and_kv(skvbc, write_set, client)
```

Comment:
Same here, I guess send_single_write_with_pre_execution_and_kv() is now dead code?

Reply:
same